dask_kubernetes/operator/daskcluster.py

```diff
@@ -80,6 +82,60 @@ def build_scheduler_service_spec(name):
     }
 
 
+def build_worker_pod_spec(name, namespace, image, n, scheduler_name):
+    return {
+        "apiVersion": "v1",
+        "kind": "Pod",
+        "metadata": {
+            "name": f"{name}-worker-{n}",
+            "labels": {
+                "dask.org/cluster-name": scheduler_name,
+                "dask.org/workergroup-name": name,
+                "dask.org/component": "worker",
+            },
+        },
+        "spec": {
+            "containers": [
+                {
+                    "name": "scheduler",
+                    "image": image,
+                    "args": ["dask-worker", f"tcp://{scheduler_name}.{namespace}:8786"],
+                }
+            ]
+        },
+    }
+
+
+def build_worker_group_spec(name, image, workers):
+    return {
+        "apiVersion": "kubernetes.dask.org/v1",
+        "kind": "DaskWorkerGroup",
+        "metadata": {"name": f"{name}-worker-group"},
+        "spec": {
+            "image": image,
+            "workers": {
+                "replicas": workers["replicas"],
```

Comment:
I was surprised to see `replicas` nested under `workers. Can we make this a top level thing under `spec`?

Reply:
Yes we can